source/getting-started/tutorials/sensors/index.md

```diff
@@ -0,0 +1,283 @@
+s
+## Experiment 1 - Ratio is too large
+
+If we increase the User Event Ratio, we can cause the control task to run at less than 10Khz. Let's increase it to 20 by putting `timing_manager_set_ratio(20)` in the `controller_init()` function.
+
+`app_controller.c`:
+```
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // set User Event Ratio
+    timing_manager_set_ratio(20);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+What does this do? We've made it so that the sensors will collect data every 20 PWM cycles, and then the scheduler will run the control task. Since our PWM frequency is still 10KHz, that means our sensors will collect data every 200us. It also means that our control task can only run once every 200us, instead of 100us like it should.
+
+![](images/tmVSI20.svg)
+
+Rebuild and run the new program, and use the command `ctrl stats print` to view the loop time (after doing `ctrl init`).
+
+```
+Task Stats:
+Loop Num:	26806 samples
+Loop Min:	193.78 usec
+Loop Max:	206.22 usec
+Loop Mean:	200.00 usec
+Loop Var:	0.02 usec
+Run Num:	26932 samples
+Run Min:	3.25 usec
+Run Max:	4.23 usec
+Run Mean:	3.41 usec
+Run Var:	0.00 usec
+```
+
+The loop mean is how much time there is between successive executions of the control task. It should be `1` / `TASK_CONTROLLER_UPDATES_PER_SEC`, but in this case it is double of that. That indicates that the control task is only running at half of `TASK_CONTROLLER_UPDATES_PER_SEC`.
+
+Making the User Event Ratio too high is one way that control tasks can be slowed down past their target `TASK_CONTROLLER_UPDATES_PER_SEC`.
+
+## Experiment 2 - Multiple sensor samples per control task
+
+By decreasing the User Event Ratio, we can cause multiple sensor samples to occur before one cycle of the control task.
+
+Let's set the User Event Ratio to `1`.
+
+`app_controller.c`:
+```
+void app_controller_init(void)
+{
+    // Enable data sampling for ADC
+    timing_manager_enable_sensor(ADC);
+    // set User Event Ratio
+    timing_manager_set_ratio(1);
+    // Register "ctrl" command with system
+    cmd_controller_register();
+}
+```
+
+What we've done now is tell the timing manager to sample the sensors every `1` PWM cycle.
+
+![](images/tmVSI1.svg)
+
+The sensors sample every PWM cycle, but the control tasks do not run every cycle. Remember that the control tasks can only run directly following a sensor sampling, but that doesn't mean that the control task always runs after every sensor sampling. In this way, the User Event Ratio can only slow down the rate of control tasks, not speed them up.
```

Comment:
"The Timing Manager triggers the sensors to sample every PWM cycle"

The paragraph explanation here seems good to me.